resources/ts/Components/ActivationDialog.tsx

```diff
@@ -81,6 +84,31 @@ type Props = {
   email: string;
 };
 
+type Address = {
+  street: string;
+  zip: string;
+  city: string;
+  country: string;
+};
+
+type Organization = {
+  name: string;
+  invoiceEmail: string;
+  vat: string;
+  address: Address;
+};
+
+type InitialValues = {
+  organization: Organization;
+  organizers: Organizer[];
+  coupon: string;
+};
+
+type Organizer = {
+  name: string;
+  id: string;
+};
```

Comment:
We have a few of these in `resources/ts/types/Organization.ts` I'd perhaps move the missing ones there too and reuse them